src/transformers/modeling_utils.py

```diff
@@ -404,6 +404,7 @@ class PreTrainedModel(nn.Module, ModuleUtilsMixin, GenerationMixin):
 
         - **base_model_prefix** (:obj:`str`) -- A string indicating the attribute associated to the base model in
           derived classes of the same architecture adding modules on top of the base model.
+        - **_is_parallelizable** (:obj:`bool`) -- A flag indicating whether this model supports model parallelization.
```

Comment:
A private flag should not appear in the public documentation.

Reply:
ah, right! so put the doc for the property then?

that's why it is no longer a hotfix, but it seems to be fine - only one user has filed an issue about using a non-working `--model_parallel` so far.

Reply:
Why not have just this class attribute be public and no property?

Reply:
I thought it was cleaner since it should be read-only, but it's fine as non-property. changed. 

I don't think we have a "way" so that's why I'm never sure when something should be a property or a public attribute.

thank you for the feedback/ideas, @sgugger.

Reply:
TBH `base_model_prefix` is the same, it should be read-only in theory but we have it as a simple class attribute, so let's stay simple for this new attribute too :-)